### PR TITLE
Changes to get Windows version, installer & AppVeyor

### PR DESCRIPTION
**Pull request by xfred81**
Following our previous PR, I try to get a more clean PR there.
(I'd need some extensive tutorial about github :))

### CODE REVIEW

**Comment by xfred81** [comment]
Hm... I've missed something as the binary is generated but the installation wizard is not build. Some lines are missing in appveyor.yml, sorry for that. I'll try to add missing lines ASAP.

**Comment by xfred81** [comment]
It should be OK now.

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    c:\msys64\usr\bin\bash -lc "cd /c/projects/grisbi-src/ && ./autogen.sh"
+
+
+    c:\msys64\usr\bin\bash -lc "cd /c/projects/grisbi-src && ./configure --prefix /c/projects/grisbi-inst/"
+    
+    
+    c:\msys64\usr\bin\bash -lc "cd /c/projects/grisbi-src && v=$(grep PACKAGE_VERSION config.h | cut -f2 -d '\"'); powershell.exe -command \"Update-AppveyorBuild -Version '$v-B%APPVEYOR_BUILD_NUMBER%'\""
+
+
+    c:\msys64\usr\bin\bash -lc "cd /c/projects/grisbi-src && make -j 2"
```
Comment (LudovicRousseau) [inline appveyor.yml]: You can use "make -j", with no number, to use the maximum of parallelism.
Unless you know that 2 is the best value here.

Comment (xfred81) [reply appveyor.yml]: 2 is the best value, indeed: https://www.appveyor.com/docs/build-environment/

**Comment by xfred81** [comment]
I open a new PR to propose 32-bit and 64-bit ports.

**Pull request closed without merging**